kpatch-build/kpatch-build

```diff
@@ -1149,6 +1155,7 @@ else
 	rm -f checksum.tmp
 	"$TOOLSDIR"/create-kpatch-module "$TEMPDIR"/patch/tmp_output.o "$TEMPDIR"/patch/output.o 2>&1 | logger 1
 	check_pipe_status create-kpatch-module
+	test "$rc" -ne 0 && die "create-kpatch-module: exited with return code: $rc"
```

Comment:
See note in review summary about pulling this out into its own pull request.  Also, just a style nit, but the rest of the script uses `[[ $rc -ne 0 ]] && ...` type syntax.

Reply:
ok sure. I will send out a separate pull request for this